cloud/lxc/lxc_container.py

```diff
@@ -448,7 +448,7 @@
         'zfs_root'
     ],
     'btrfs': [
-        'lv_name', 'vg_name', 'thinpool', 'zfs_root'
+        'lv_name', 'vg_name', 'thinpool', 'zfs_root', 'fs_type', 'fs_size'
```

Comment:
Quick question here but does it really need to NOT have storage type and size when using a btrfs container?  I've not seen that though admittedly I've not tested btrfs in some time. If so, the change lgtm but I'd like to know why this change is required first.